BackwardConfig.cmake

```diff
@@ -24,10 +24,17 @@
 # OPTIONS
 ###############################################################################
 
+if (APPLE)
```

Comment:
any way we can avoiding setting this here and let autodetection do the work?

Reply:
Strange. I had thought unwind wasn't working on OS X, but it does. So I added in support, which let's me remove this.